packages/gatsby/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path')
```

Comment:
Why are you compiling it with webpack? You should use babel like the rest of Gatsby codebase.

Reply:
This was originally an attempt to bring our own React without having to worry about the user's version. For now we will instead look to error on old React versions when the recipes command is run.